hack/prow/prow.mk

```diff
@@ -16,4 +15,11 @@ integration-prow-kvm-docker-linux-x86-64:
 	cp -f "${GCE_SSH_PRIVATE_KEY_FILE}" ~/.ssh/google_compute_engine
 	cp -f "${GCE_SSH_PUBLIC_KEY_FILE}" ~/.ssh/google_compute_engine.pub
 	GOTOOLCHAIN=auto go build -C ./hack/prow/minitest -o $(PWD)/out/minitest .
-	./out/minitest  --deployer boskos --tester kvm-docker-linux-amd64-integration --config hack/prow/kvm.json
\ No newline at end of file
+	./out/minitest  --deployer boskos --tester kvm-docker-linux-amd64-integration --config hack/prow/kvm.json
+
+push-kubernetes-bootcamp:
```

Comment:
how about naming it prow so all prow things have a prow in their name


push-prow-kubernetes-bootcamp:

Reply:
we can't change it here because it is referenced in test-infra